### DISCUSSION TITLE
Octopus Energy integration wrapped 2024

### DISCUSSION THREAD

**Discussion by BottlecapDave** [BottlecapDave/HomeAssistant-OctopusEnergy#1141] [Announcements]
With the year coming to a close, I thought I'd share some stats

* Over 96 hours of known dev time has gone into features and bug fixes (with at least another 9 hours in the pipeline). I didn't start tracking this until june and I still forget to log it from time to time, so I'm sure it's a lot more this year alone!
* 41 versions have been released, which have included 4 major versions
* Went from 400 to almost 650 stars on GitHub
* Went from 3000 known installs to around 5400 known installs (welcome if that was you!)

I still find it amazing how many people use this integration :)

**Comment by res2cpu** [comment]
Thank you for the work you do on this!

I use it to run my battery charging to load shift off to my batteries which has saved me nearly £750 in the last year.

**Reply by BottlecapDave** [reply]
Glad to here it helped you save some money :)

**Comment by crashedash32** [comment]
thank you so much for your efforts on this, it is the heart of my system. be lost without it.

**Comment by ragg987** [comment]
Thanks for all your work on squashing bugs, adding features and good documentation. My energy cost optimisation in HA depends on this add-on, in particular I'm heavily reliant on target sensors. I dread the day Octopus no longer remain a viable energy supplier for me - the thought of having to lose all this capability...

Hopefully Octopus are contributing to this add-on - it certainly helps in keeping me a "captive customer".

**Reply by BottlecapDave** [reply]
Thank you for the kind words. External contributions are minimal but OE help with questions from time to time. 

I've been thinking about pulling some features out of this integration (but not removing) so people with other suppliers could potentially take advantage of some of the features. So what this space if you ever do move on from OE.

**Reply by ragg987** [reply]
Might be worth seeing if OE willing to contribute as you are helping lock-in my custom with them. There are likely to be a bunch of users, like me, who are heavily invested thanks to HA and your add-on and are not likely to switch suppliers in a hurry. Whilst Octopus are currently attractive in terms of pricing, other like EoN are also innovating so the competition landscape is evolving...

**Reply by BottlecapDave** [reply]
They've been toying with HA with their beta product the home pro. I've asked what their future plans/ intentions are, but got radio silence on that matter. This is why I've been thinking about pulling bits out of the integration so that others can benefit as well as the scenario that OE produce their own integration but not with all of the bells and whistles.
